tests/implode_hamlet_256.bin

```diff
@@ -0,0 +1,2 @@
+
#67h���<D��������P!T�oՖK�Ѻt˺[V�Y�E��}��[� ��m˶,]� ��y5-[�i�7ȩhî-;nٰr���Mmڹ ˼7�7�V.H�h˂�;����6�[�e�����·��^�m�tM&Y�
c��t?c�����
��o8ٚ���>�@k6��|�t����~���A�l����_�
```

Comment:
Git thinks this is a text file; to tell it otherwise, add `*.bin binary` to `.gitattributes`. Also, move this to `tests/data`.

Reply:
moved to test/data - all files in test/data are binary.